src/components/App.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
+
+export interface HelloProps { greet: string; company: string };
```

Comment:
Make a separate folder for interfaces. We will follow structure like,
```
- src
   - index.tsx
   - app.tsx
   - interfaces (contains interfaces for props and state or any other related only to app.tsx)
      - IAppProps.ts
      - IAppState.ts
   - home-page
      - interfaces
      - HomePageComponent
      - homePageStyle.css
```
So create a new folder for interfaces and move this interface to that folder.

Reply:
Okay, Will do, Thanks!